src/Raven.Server/ServerWide/Commands/UpdateUnusedDatabaseIdsCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Raven.Client.ServerWide;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.ServerWide.Commands
+{
+    public class UpdateUnusedDatabaseIdsCommand : UpdateDatabaseCommand
+    {
+        public HashSet<string> UnusedDatabaseIds;
+
+        public UpdateUnusedDatabaseIdsCommand()
+        {
+            
+        }
+        public UpdateUnusedDatabaseIdsCommand(string database, HashSet<string> list, string raftId) : base(database, raftId)
+        {
+            UnusedDatabaseIds = list;
+        }
+
+        public override string UpdateDatabaseRecord(DatabaseRecord record, long etag)
+        {
+            record.UnusedDatabaseIds = UnusedDatabaseIds;
```

Comment:
Maybe we should make it more atomic operation, like adding/removing database ids

Reply:
We are very unlikely to have to do this very often, I don't think it matters